### DISCUSSION TITLE
I get an error trying to import the demo app to Sense QMC

### DISCUSSION THREAD

**Discussion by rumen-vasilev** [ptarmiganlabs/butler#519] [General]
Can you help me, please?

![image](https://user-images.githubusercontent.com/83919259/182791464-1f4cc78b-3baf-4ad5-8da7-2bfc5f7c8174.png)

**Comment by mountaindude** [comment]
Looks like that QVF file may be corrupt somehow. 
I'll take a look during coming days/week.

What version of Qlik Sense are you trying to import the app into?

**Reply by rumen-vasilev** [reply]
Hi, thanks for your support. It's Qlik Sense Sense 2022 February Patch 2.

**Comment by mountaindude** [comment]
@rumen-vasilev 

This has been fixed and the 8.4 demo app can now be downloaded directly from the repo at GitHub, or cloned to local computer and used from there.

The bug was tracked in #533.
